portfolio/src/main/java/com/google/sps/servlets/ChartDataServlet.java

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/destination-data")
+public class ChartDataServlet extends HttpServlet {
+  private Map<String, Long> colorVotes = new HashMap<>();
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Votes");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    Map<String, Long> updateVote = new HashMap<>();
+    for (Entity entity : results.asIterable()) {
+      String trip = (String) entity.getProperty("destination:");
+      long currentVotes = updateVote.containsKey(trip) ? updateVote.get(trip) : 0;
+      long pls = (long) entity.getProperty("total votes:");
+      
+      updateVote.put(trip,pls+currentVotes);
```

Comment:
[Style](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace): Horizontal whitespace appears "on both sides of any binary or ternary operator"
```suggestion
      updateVote.put(trip,pls + currentVotes);
```